<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>聚火 选股小助手</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,7">
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_basic">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>157</width>
         <height>641</height>
        </rect>
       </property>
       <attribute name="label">
        <string>基础功能</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="page_extend">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>157</width>
         <height>641</height>
        </rect>
       </property>
       <attribute name="label">
        <string>扩展功能</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page_stocklist">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableWidget" name="tableWidget"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_filterinfo"/>
      <widget class="StockChart" name="page_stockchart"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StockChart</class>
   <extends>QWidget</extends>
   <header>StockChart.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
